daprdocs/content/en/developing-applications/authenticating/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+type: docs
+title: "Authenticating to services"
+linkTitle: "Authenticating to services"
+weight: 10
```

Comment:
I recommend moving this file/directory to ./integrations and updating the weight to be 3000

Reply:
@orizohar what are your thoughts on this suggest organization?

Reply:
Great, thanks for the review! I'll wait until we hear from @orizohar before I make any further changes here.

Reply:
I agree, this is better placed under integrations